src/main/java/com/amogoscode/groupe/ebankingsuite/universal/ApiResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.amogoscode.groupe.ebankingsuite.universal;
+
+import lombok.EqualsAndHashCode;
```

Comment:
remove non-necessary imports

Reply:
changes have been pushed in the new commit